SEO/Models/CrawlInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MongoDB.Bson;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace SEO.Models
+{
+    public class CrawlInfo
```

Comment:
the response will not be related only to crawl info. it will contain other data related to title and body. therefore the class name should be renamed. Correct naming is important.
Propose to PageRenderingInfo